foundation/foundation-archetype/src/main/resources/archetype-resources/__rootArtifactId__-deploy/src/main/resources/values-ci.yaml

```diff
@@ -3,7 +3,3 @@ spec:
     valueFiles:
       - values.yaml
       - values-ci.yaml
-
-global:
```

Comment:
Q: why remove this?

Reply:
the same values are already defined in the values.yaml